.gitignore

```diff
@@ -0,0 +1,3 @@
+test.md 
+
```

Comment:
[ OPTIONAL ]Please add all unnecessary files and folders like `vscode` to gitignore.